src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -378,66 +253,74 @@ remove the mouse event listeners to the document.
 
 Now that we have the start and end events taken care of we can actually respond to the touch events.
 
-#### Get and Store Touch Event Coordinates
-
 For any of the start and move events, you can easily extract `x` and `y`
 from an event.
 
-The following code snippet checks whether the event is from a touch event by looking for `targetTouches`, if it is then it extracts the `clientX` and `clientY` from the first touch. If the event is a mouse or pointer event then we extract `clientX` and `clientY` directly from the event itself.
+The following code snippet checks whether the event is from a `TouchEvent` by checking if `targetTouches` exists, if it does, then it extracts the `clientX` and `clientY` from the first touch.
+If the event is a `PointerEvent` or `MouseEvent` we extract `clientX` and `clientY` directly from the event itself.
 
 <pre class="prettyprint">
 {% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-2.html" region_tag="extract-xy" adjust_indentation="auto" %}
 </pre>
 
-Each touch event has three lists containing touch data
-(see also [Touch lists](#touch-lists)):
+A TouchEvent has three lists containing touch data:
 
 * `touches`: list of all current touches on the screen, regardless of DOM element they are on.
 * `targetTouches`: list of touches currently on the DOM element the event is bound to.
 * `changedTouches`: list of touches which changed resulting in the event being fired.
 
-In most cases, `targetTouches` gives you everything you need.
+In most cases, `targetTouches` gives you everything you need and want. (For
+more info on these lists see [Touch lists](#touch-lists)).
 
-#### requestAnimationFrame
+#### Use requestAnimationFrame
 
 Since the event callbacks are fired on the main thread, we want to run as
-little code as possible in the callback to keep our frame rate high,
-preventing jank.
+little code as possible in the callbacks for our events, keeping our frame
+rate high and preventing jank.
 
-Use `requestAnimationFrame` to change the UI in response to
-an event. This gives you an opportunity to update the UI when the browser is intending to draw a frame and will help you move some work out of your callback.
+Using `requestAnimationFrame` we have an opportunity to update the UI when
+the browser is intending to draw a frame and will help us move some work out
+of our event callbacks.
 
```

Comment:
This sentence needs some work. I recommend this:

The `requestAnimationFrame()` method lets us update the UI when the browser is drawing a frame and helps us move work out of our callbacks.